src/uwtools/drivers/cdeps.py

```diff
@@ -26,7 +26,7 @@ def atm(self):
         """
         Create data atmosphere configuration with all required content.
         """
-        yield self._taskname("data atmosphere configuration")
+        yield self.taskname("data atmosphere configuration")
```

Comment:
Here and in many files that follow, change `_taskname` to `taskname`. In some places, this leads to moving a function in a module. Anything involving "taskname" is probably just this, so you can pattern-match and ignore once you see the pattern.